ansible/roles/operator-pipeline/templates/openshift/pipelines/community-hosted-pipeline.yml

```
@@ -327,6 +327,34 @@ spec:
         - name: github_token_secret_key
           value: "$(params.github_token_secret_key)"
 
+    # merge PR
+    - name: merge-pr
+      when:
+        - input: "$(tasks.get-ci-reviewer.results.author_is_reviewer)"
+          operator: notin
+          values: ["false"]
```

Just for the sake of the principles of defensive programming I would use the opposite:

```suggestion
        - input: "$(tasks.get-ci-reviewer.results.author_is_reviewer)"
          operator: in
          values: ["true"]
```

but it's not a big deal here.

---

Updated.